ansible/configs/sap-hana/print_info_osp.yml

```diff
@@ -0,0 +1,45 @@
+- name: Set agnosticd user info messages
+  agnosticd_user_info:
+    msg: "{{ item }}"
+  loop: >-
+    {{
+      [
+        "You can access your bastion via SSH:",
+        "",
+        "ssh cloud-user@bastion-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}",
+        "",
+        "",
+        "Make sure you use the following RSA key while connecting to the Bastion Host:",
+        ""
+      ]
+      + lookup('file', '{{ output_dir }}/{{ guid }}_infra_ssh_key.pem').splitlines()
+      + (
+      [
+        "",
+        "",
+        "You can access your Ansible Tower instance via HTTPS:",
+        "",
+        "https://tower-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}",
+        "",
+        "",
+        "Make sure you are using the following credentials:",
+        "",
+        "Username: admin",
+        "Password: {{ tower_password }}"
+      ] if tower_instance_count > 0 else []
+      )
+    }}
+
+- name: Set agnosticd user info data for bastion
+  agnosticd_user_info:
+    data:
+      bastion_ssh_command: "ssh cloud-user@bastion-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
+      bastion_ssh_key: "{{ lookup('file', '{{ output_dir }}/{{ guid }}_infra_ssh_key.pem') }}"
+
+- name: Set agnosticd user info data for tower
+  when: tower_instance_count > 0
+  agnosticd_user_info:
+    data:
+      tower_url: "https://tower-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
+      tower_username: "admin"
+      tower_password: "{{ tower_password }}"
```

Comment:
do you really need 2 print_info.yml files, one for each cloud provider ?

Things that differ could be set in the default_vars_...  files

for example in the common print_info.yml:
```yaml
bastion_ssh_command: "ssh {{ remote_user }}@{{ bastion_public_dns_chomped }}"
```

Then 

default_vars_osp:
```yaml
bastion_public_dns_chomped: bastion-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
```